tapdb/assets_store.go

```diff
@@ -873,12 +873,38 @@ func (a *AssetStore) constraintsToDbFilter(
 				Valid: true,
 			}
 		}
+
+		if query.MaxAmt != 0 {
+			assetFilter.MaxAmt = sql.NullInt64{
```

Comment:
nit: can use `sqlInt64(query.MaxAmt)` (and change it above for the `MinAmt` as well).